packages/contracts-bedrock/contracts/L2/L1Block.sol

```diff
@@ -74,5 +83,49 @@ contract L1Block is Semver {
         batcherHash = _batcherHash;
         l1FeeOverhead = _l1FeeOverhead;
         l1FeeScalar = _l1FeeScalar;
+
+        _setL1BlockHash(_number, _hash);
```

Comment:
Since we are only using this function once isn't it cheaper to save the function call and use the assembly here inline instead?